packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1295,12 +1295,12 @@ export default class Meeting extends StatelessWebexPlugin {
      */
     this.remoteMediaManager = null;
 
-    this.localAudioStreamMuteStateHandler = (muted: boolean) => {
-      this.audio.handleLocalStreamMuteStateChange(this, muted);
+    this.localAudioStreamMuteStateHandler = () => {
+      this.audio.handleLocalStreamMuteStateChange(this);
```

Comment:
- From here: [feat: separate API for enabled and muted states](https://github.com/webex/webrtc-core/pull/72/files#diff-d16518f95fd5fb1272a6455c8d066afdb87254d823088f880ff255a004636467R18) I can see that the event still emits the boolean for `muted`

Reply:
It is simply because we aren't using the parameter in this particular case. Inside `this.audio.handleLocalStreamMuteStateChange`, we manually check both `systemMuted` and `userMuted` states of the stream. The `muted` parameter here would only give us one of them... and it could be either  `systemMuted` and `userMuted`, since this handler is triggered both on `UserMuteStateChange` and `SystemMuteStateChange`.

Reply:
Alright. Got it. Thanks @brycetham